go.mod

```diff
@@ -325,6 +325,8 @@ replace k8s.io/client-go => k8s.io/client-go v0.25.9
 
 replace knative.dev/pkg => knative.dev/pkg v0.0.0-20230502134655-db8a35330281
 
+replace github.com/tektoncd/pipeline => github.com/tektoncd/pipeline v0.47.4
```

Comment:
Why do we need that replace ?

Reply:
by default in these cli dep, 0.47.0 is coming.

here i have bumped to make it 0.47.4, or i think i can add directly to rquire too :D

but this added for bundle kind field which affecting tkn pac too

Reply:
oh and it's for 1.11 branch, I see.